### PR TITLE
fix 'dotnet build' command error for net35

### PR DESCRIPTION
**Pull request by hanabi1224**
error MSB3644: The reference assemblies for framework ".NETFramework,Version=v3.5" were not found. To resolve this, install the SDK or Targeting Pack for this framework version or retarget your application to a version of the framework for which you have the SDK or Targeting Pack installed. Note that assemblies will be resolved from the Global Assembly Cache (GAC) and will be used in place of reference assemblies. Therefore your assembly may not be correctly targeted for the framework you intend.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+test_script:
+  - cmd: dotnet test -c Release src\Senparc.NeuChar.Tests\
+artifacts:
+  - path: '**\*.nupkg'
+nuget:
+  disable_publish_on_pr: true
+#deploy:
+#  provider: NuGet
+#  api_key:
+#    secure: <Your secret>
```
Comment (hanabi1224) [inline appveyor.yml]: api_key加上的话再把deploy这里反注释掉就可以自动发布nuget了

**Comment by JeffreySu** [comment]
感谢提交！master分支只允许从 Develop 分支 PR 并 Review，麻烦 PR 到 Develop 分支，有部分代码还需要修改。

**Comment by hanabi1224** [comment]
@JeffreySu 好的，那最好在设置里把默认branch改成 Develop : )

**Comment by hanabi1224** [comment]
@JeffreySu  Created a new PR: https://github.com/Senparc/NeuChar/pull/35

**Comment by JeffreySu** [comment]
@hanabi1224  因为很多人打开之后直接会点下载，所以用Develop做默认分支可能会导致一些混乱，有时候里面会有一些还未完善的代码，我暂时还是会保持master做默认分支，感谢建议！

**Pull request merged**